chart/open-feature-operator/values.yaml

```diff
@@ -8,13 +8,23 @@ sidecarConfiguration:
   image:
     # these fields must remain in the same order, renovate uses a regex to update the tag value
     repository: "ghcr.io/open-feature/flagd"
-    tag: v0.4.4
+    tag: v0.4.5
   providerArgs: ""
   envVarPrefix: "FLAGD"
   defaultSyncProvider: kubernetes
   evaluator: json
   logFormat: "json"
   probesEnabled: true
+  debugLogging: false
+
+kubeProxyConfiguration:
+  port: 8015
+  metricsPort: 8016
+  image:
+     # these fields must remain in the same order, renovate uses a regex to update the tag value
+    repository: "ghcr.io/open-feature/kube-flagd-proxy"
+    tag: v0.1.2
```

Comment:
add to renovate following https://github.com/open-feature/open-feature-operator/issues/415